### DISCUSSION TITLE
Windows CI编译的版本与Release中的版本有哪些差异？

### DISCUSSION THREAD

**Discussion by wesley800** [c0re100/qBittorrent-Enhanced-Edition#486] [Q&A] [answered]
qbittorrent.exe文件大小有比较大的差距（CI编译出来是20M左右，Release版本是30M），是否因为Release版本额外启用了某些功能？

**Comment by c0re100** [comment, marked as answer]
Release版本是靜態編譯，而CI不是
所以你會看到CI版本是連帶一堆Qt的dll

**Reply by wesley800** [reply]
感谢光速回复！原来是因为我覆盖安装到CI版本的目录，结果留下了额外的dll没有被删除导致误判，十分抱歉。

**Comment by yeezylife** [comment]
最新的4.6.4 CI编译的windows版本，一旦同时“继续”多个种子文件就会闪退（选取大约6个或者以上暂停的种子，点继续就会闪退。。）
